scheduler/src/cook/kubernetes/api.clj

```diff
@@ -568,20 +580,26 @@
               (log/error e "In" compute-cluster-name "compute cluster, error during event watch"))))
         (finally
           (.close watch)
-          (initialize-event-watch api-client compute-cluster-name all-pods-atom))))))
+          (initialize-event-watch compute-cluster))))))
 
 (defn initialize-event-watch
   "Initializes the event watch"
-  [^ApiClient api-client compute-cluster-name all-pods-atom]
+  [{:keys [state-atom] compute-cluster-name :name :as compute-cluster}]
   (log/info "In" compute-cluster-name "compute cluster, initializing event watch")
   (let [{:keys [reconnect-delay-ms]} (config/kubernetes)
         tmpfn (fn []
                 (try
                   (log/info "In" compute-cluster-name "compute cluster, initializing event watch helper")
-                  (initialize-event-watch-helper api-client compute-cluster-name all-pods-atom)
+                  (initialize-event-watch-helper compute-cluster)
                   (catch Exception e
-                    (log/error e "Error during event watch initial setup of looking at events for" compute-cluster-name
-                               "and sleeping" reconnect-delay-ms "milliseconds before reconnect")
+                    (let [state @state-atom]
+                      (if (= state :deleted)
+                        (log/info
+                          e "In" compute-cluster-name "compute cluster, cannot create event watch (cluster is deleted)"
```

Comment:
can we still leave the message saying that there's a sleep?

Reply:
👍🏼 Done